src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static Random random = new Random();
+
+    private static ColorSupplier ballColor = new ColorSupplier();
+
+    public static Ball getRandomBall() {
+        int ballNumber = random.nextInt(100);
```

Comment:
use constants instead of numbers
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)

Reply:
not fixed